cypress/integration/Switch.spec.ts

```diff
@@ -0,0 +1,58 @@
+import * as h from '../helpers';
+
+const getSwitch = () => {
+  return cy.get(`[type="checkbox"]`);
+};
+
+describe('Non-disabled Switch', () => {
+  before(() => {
+    h.stories.visit();
+  });
+  ['Default', 'Alert', 'Error'].forEach(story => {
+    context(`Given the '${story}' story is rendered`, () => {
+      beforeEach(() => {
+        h.stories.load('Components|Inputs/Switch/React/Top Label', story);
+      });
+
+      it('should not have any axe errors', () => {
+        cy.checkA11y();
+      });
+
+      context('when the tab key is pressed', () => {
+        beforeEach(() => {
+          cy.tab();
+        });
+        it('should be the focused item on the page', () => {
+          getSwitch().should('have.focus');
```

Comment:
This specification is a little weird because hitting the tab key in this instance will give focus to the first focusable element on the page. I think we are saying the Switch should be focusable as per accessibility

Reply:
Removed